spec/index.html

```diff
@@ -559,27 +556,28 @@ <h3>Triples</h3>
       |p| is called the <dfn>predicate</dfn> of the triple, and
       |o| is called the <dfn>object</dfn> of the triple.</p>
 
-    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF triple</a> that is
-      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+    <p>A <dfn>graph term</dfn> is an <a>RDF graph</a> that is
+      used as the <a>subject</a> or <a>object</a> of a <a>triple</a>.</p>
 
     <p class="note">By the given definitions,
       the <a>subject</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, or a <a>quoted triple</a>;
+      a <a>blank node</a>, or a <a>graph term</a>;
       the <a>predicate</a> of an <a>RDF triple</a> may only be an <a>IRI</a>;
       the <a>object</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, a <a>literal</a> or a <a>quoted triple</a>.</p>
+      a <a>blank node</a>, a <a>literal</a> or a <a>graph term</a>.</p>
 
-    <p class="note">The definition of <a>quoted triple</a> is recursive.
-      That is, a <a>quoted triple</a> can itself have a
-      <a>subject</a> or <a>object</a> component which is another <a>quoted triple</a>.
-      However, by this definition, cycles of <a>quoted triples</a> cannot be created.</p>
+    <p class="note">The definition of <a>graph term</a> is recursive.
+      That is, a <a>graph term</a> can itself contain
+      triples where the <a>subject</a> or <a>object</a> components
+      are <a>graph terms</a>.
```

Comment:
Reestablishing @TallTed's [change suggestion](https://github.com/w3c/rdf-concepts/pull/67#discussion_r1361298265): 
```suggestion
      That is, a <a>graph term</a> can itself contain
      triples where either or both the <a>subject</a> and/or <a>object</a> component
      is a <a>graph term</a>.
```